src/metabase/models/dashboard.clj

```diff
@@ -247,6 +255,8 @@
                                                     :model/DashboardTab :dashboard_id dashboard-id)
         {:keys [old->new-tab-id]} (dashboard-tab/do-update-tabs! dashboard-id current-tabs (:tabs serialized-dashboard))
         serialized-cards          (cond->> serialized-cards
```

Comment:
Can we name this `serialized-dashcards` while we're at it
```suggestion
        serialized-dashcards          (cond->> serialized-dashcards
```

Reply:
yes 84f8fdbabc